app/views/partials/_similar_artists.html.haml

```diff
@@ -0,0 +1,24 @@
+.mui-container
+    #similar-artists.mui-row.artists-list
+        .mui-col-lg-12
+            %h3.artist-heading Similar Artists.
```

Comment:
Do you need the dot ;)

Reply:
It was in the original design. Not sure.